apps/portals/elportal/src/config/synapseConfigs/handleParticipantsToFiles.tsx

```diff
@@ -41,7 +41,7 @@ export const handleSelectedParticipantsToFiles = async (
   event: CustomControlCallbackData,
 ) => {
   // add filter for files perspective, to show files associated to the selected participants only.
-  const idColIndex = event.data?.columnModels?.findIndex(
+  const idColIndex = event.queryMetadata?.columnModels?.findIndex(
```

Comment:
`data` was changed to `queryMetadata` to make it more clear that row data is not included.